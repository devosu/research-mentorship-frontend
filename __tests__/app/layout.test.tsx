// ./__tests__/app/layout.test.tsx
//
// Snapshot test for the gloabl layout, DO NOT update,
// unless you know what you are doing.

// Testing essential imports.
import { describe, expect, it } from "@jest/globals";
import { render } from "@testing-library/react";

// Local imports.
import RootLayout from "@app/layout";

// Test suite.
describe("The RootLayout", () => {
  it("correctly renders children", () => {
    const { container } = render(
      <RootLayout>
        <div>Test Child Text</div>
      </RootLayout>,
    );
    expect(container).toMatchSnapshot();
  });
});
