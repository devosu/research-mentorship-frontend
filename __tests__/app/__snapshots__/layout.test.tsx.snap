// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The RootLayout correctly renders children 1`] = `
<div>
  <html
    lang="en"
  >
    <body>
      <div>
        Test Child Text
      </div>
    </body>
  </html>
</div>
`;
